src/settings/Settings.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import SelectWithLabelFormGroup from '../components/input/SelectWithLableFormGroup'
+import i18n, { resources } from '../i18n'
+import useTitle from '../page-header/useTitle'
+
+const Settings = () => {
+  const { t } = useTranslation()
+  useTitle(t('settings.label'))
+
+  // Language section
+  const languageOptions = Object.keys(resources)
+    .map((abbr) => ({
+      label: resources[abbr].name,
+      value: abbr,
+    }))
+    .sort((a, b) => (a.label > b.label ? 1 : -1))
```

Comment:
we have lodash as a dependency to this project, so we can make use of it to do the sorting. https://lodash.com/docs/#sortBy

Reply:
✅ in 8f2bc88